### PR TITLE
streams.part2.exercise1 (Korobovskii)

### PR DESCRIPTION
**Pull request by ksil29**

### CODE REVIEW

File: src/test/java/streams/part2/exercise/Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    @SuppressWarnings("Duplicates")
+    public void groupPersonsByFirstPositionUsingToMap() {
+        List<Employee> employees = getEmployees();
+
+        // реализация
+        Function<Employee, Set<Person>> employeeToSetPersonFunction = e ->
+            new HashSet<>(Collections.singletonList(e.getPerson()));
+
+        BinaryOperator<Set<Person>> mergeSets = (left, rigth) -> {
+            left.addAll(rigth);
```
Comment (shaart) [inline src/test/java/streams/part2/exercise/Exercise1.java]: typo at word "right"

File: src/test/java/streams/part2/exercise/Exercise1.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+    @Test
+    @SuppressWarnings("Duplicates")
+    public void groupPersonsByFirstPositionUsingGroupingByCollector() {
+        List<Employee> employees = getEmployees();
+
+        // реализация
+
+
+        Map<String, Set<Person>> result =
```
Comment (shaart) [inline src/test/java/streams/part2/exercise/Exercise1.java]: Blank lines (121-122)

**Review by shaart — APPROVED** [review]
There is a typo. Anyway it's fine

File: src/test/java/streams/part2/exercise/Exercise1.java (hunk 2 above)
Comment (ksil29) [reply src/test/java/streams/part2/exercise/Exercise1.java]: Done

File: src/test/java/streams/part2/exercise/Exercise1.java (hunk 1 above)
Comment (ksil29) [reply src/test/java/streams/part2/exercise/Exercise1.java]: Done

**Review by Meissshu — APPROVED** [review]

**Pull request merged**